inc/template-tags.php

```diff
@@ -192,7 +203,20 @@ function twentytwenty_get_post_meta( $post_id = null, $location = 'single-top' )
 
 	// Get the post meta settings for the location specified.
 	if ( 'single-top' === $location ) {
-
+		/**
+		 * Filters post meta info visibility
+		 *
+		 * Use this filter to hide post meta information like Author, Post date, Comments, Is stikcy status
```

Comment:
There is a tiny typo here:   _Is stikcy status_

Reply:
Typo fixed.